xlsxwriter/worksheet.py

```diff
@@ -378,51 +410,31 @@ def write(self, row, col, *args):
         if token is None:
             return self.write_blank(row, col, *args)
 
+
+        # avoid isinstance as much as possible (because it's slow)
```

Comment:
Note. All comments in XlsxWriter should be properly cased and punctuated sentences.